### PR TITLE
Fix README.dev

### LINKED ISSUE #539: Use Prettier for Markdown formatting
### Discussed in https://github.com/TulipaEnergy/TulipaEnergyModel.jl/discussions/536

<div type='discussions-op-text'>

<sup>Originally posted by **gnawin** February 26, 2024</sup>
### What and Why
Currently we have markdown lint for Markdown formatting, we want to use Prettier instead because it includes a table formatter. This means:

- [ ] Delete markdown lint (both in vscode and in pre-commit)
- [ ] Install Prettier (both in vscode and in pre-commit)
- [ ] Update README.dev 

### Possible Drawbacks
Prettier has opinionated Markdown formatting, but does not allow customization

### PR DESCRIPTION
**Pull request by gnawin**
# Pull request details

## Describe the changes made in this pull request

For some reason, some of the indentation in PR #552 went wrong. This PR fixed it.

## List of related issues or pull requests

Closes #539 

## Collaboration confirmation

As a contributor I confirm

-   [x] I read and followed the instructions in README.dev.md
-   [x] The documentation is up to date with the changes introduced in this Pull Request (or NA)
-   [x] Tests are passing
-   [x] Lint is passing

### CODE REVIEW

File: README.dev.md [hunk 1]
```diff
@@ -49,27 +49,27 @@ To contribute to TulipaEnergyModel.jl, you need the following:
     formatting. To install it, open Julia REPL, for example, by typing in the
```
Comment (clizbe) [inline README.dev.md:49]:
```suggestion
    formatting. 
    To install it, open Julia REPL, for example, by typing in the
```

Comment (clizbe) [reply README.dev.md:49]: Will this make it go to a new line?
Otherwise looks good.

Comment (clizbe) [reply README.dev.md:49]: Yes that's what I was trying to say. :)

**Comment by datejada** [comment]
Hi @gnawin, please also consider adding the changes suggested here: https://github.com/TulipaEnergy/TulipaEnergyModel.jl/discussions/536#discussioncomment-8836734

**Review by clizbe — APPROVED** [review]
Great!

**Pull request merged**